ui/lib/core/stories/empty-state.md

```diff
@@ -18,7 +20,7 @@ encounters a state that would usually be blank.
 
 **See**
 
-- [Uses of EmptyState](https://github.com/hashicorp/vault/search?l=Handlebars&q=EmptyState)
-- [EmptyState Source Code](https://github.com/hashicorp/vault/blob/master/ui/app/components/empty-state.js)
+- [Uses of EmptyState](https://github.com/hashicorp/vault/search?l=Handlebars&q=EmptyState+OR+empty-state)
+- [EmptyState Source Code](https://github.com/hashicorp/vault/blob/master/ui/lib/core//addon/components/empty-state.js)
```

Comment:
looks like this URL is broken -- did you remember to include the name of the addon, i.e. `yarn gen-story-md empty-state core`?

Reply:
Ran it again and it did change it.  Thanks for checking!